docs/writing-policies/go/07-automate.md

```diff
@@ -1,60 +1,59 @@
 ---
-sidebar_label: "GitHub Action Integration"
-title: ""
+sidebar_label: GitHub Actions
+title: Integrating with GitHub Actions
+description: Integrating with GitHub actions when developing policies for Kubewarden in Go.
+keywords: [kubewarden, kubernetes, github, integration]
+doc-type: [tutorial]
+doc-topic: [kubewarden, writing-policies, golang, github-action-integration]
+doc-persona: [kubewarden-developer]
 ---
 
-# Automations
+## Automation
 
-This section describes how we can use GitHub Actions to automate as many tasks
-as possible.
+This section describes how you can use GitHub Actions to automate tasks.
 
-The scaffolded project already includes all the GitHub actions you need.
-These Actions can be found in the `.github/workflows/ci.yml.template` file;
-rename it to `github/workflows.ci/yml` to enable them.
+The project scaffolding already includes all the GitHub actions you need.
+You can find the Actions in the `.github/workflows/ci.yml.template` file.
+Rename it to `.github/workflows.ci/yml` to enable them.
 
-The same principles can be adapted to use a different CI system.
+You can adapt these principles to use a different CI system.
 
 ## Testing
 
-Automation of the unit tests and of the end-to-end tests is working out of the
-box thanks to the `unit-tests` and `e2e-tests` jobs defined in
-`.github/workflows/ci.yml.template`.
+Automation of the unit tests and of the end-to-end tests works out of the box.
+It uses the `unit-tests` and `e2e-tests` jobs defined in `.github/workflows/ci.yml.template`.
 
 ## Release
 
-The scaffolded project contains a `release` job in
-`.github/workflows/ci.yml.template`.
+The project scaffolding has a `release` job in `.github/workflows/ci.yml.template`.
 
 This job performs the following steps:
 
-  * Checkout code
-  * Build the WebAssembly policy
-  * Push the policy to an OCI registry
-  * Eventually create a new GitHub Release
+- Checkout code
+- Build the WebAssembly policy
+- Push the policy to an Open Container Initiative (OCI) registry
+- Create a new GitHub Release
 
-To enable the job you need to rename it to `ci.yml` and change the value of the
-`OCI_TARGET` to match your preferences.
+To enable the job, rename it to `ci.yml` and change the value of the `OCI_TARGET` to match your preferences.
```

Comment:
This is still required. But in a different way. As the files uses our github actions, the policy author need to change the [`oci-target`](https://github.com/kubewarden/go-policy-template/blob/ffef5e548495d9688d694316a4566dcf9ec88bec/.github/workflows/release.yml#L28) input used by the action

Reply:
Adjusted, thanks.